testbook/client.py

```diff
@@ -243,7 +242,9 @@ def value(self, code: str) -> Any:
             outputs = self.inject(inject_code, pop=True).outputs
             return outputs[0].data['application/json']['value']
 
-        except ValueError:
+        except TestbookRuntimeError as te:
+            if te.nbexception is not ValueError:
```

Comment:
This looks odd?

Reply:
Yeah, it is still WIP

Reply:
What you need is an additional catch for `outputs` being empty and for `'application/json'` or `'value'` to be absent